setup.py

```diff
@@ -29,14 +29,10 @@ def find_version(*file_paths):
     's3transfer>=0.3.0,<0.4.0',
     'PyYAML>=3.10,<5.5',
     'colorama>=0.2.5,<0.4.4',
+    'rsa>=3.1.2,<4.5.0; python_version=="2.7"',
```

Comment:
This should probably be `<=` so we pull in `4.5` instead of `4.4` and are consistent with the `setup.cfg`